src/routes/v1/index.js

```diff
@@ -1,8 +1,10 @@
 import authRoutes from './auth.route';
 import userRoutes from './user.route';
+import articleRoutes from './article.route';
 
 export default app => {
   app.use('/auth', authRoutes);
   app.use('/api/v1/users', userRoutes);
   app.use('/api/v1/auth', userRoutes);
+  app.use('/api/v1/', articleRoutes);
```

Comment:
please use ```app.use('api/v1/articles', articleRoutes)``` to maintain consistency

Reply:
okay @tolumide-ng . I will do that. Thank you